extensions/gamejolt.js

```diff
@@ -721,29 +835,47 @@
       );
     };
 
-    GJAPI.DataStoreRemove = function (iStore, sKey, pCallback) {
+    /**
+     * Send remove-data request
+     * @param {number} iStore
+     * @param {string} sKey
+     * @param {function} pCallback
+     */
+    GJAPI.DataStoreRemove = (iStore, sKey, pCallback) => {
       // send remove-data request
       GJAPI.SendRequest("/data-store/remove/?key=" + sKey, iStore, pCallback);
     };
 
-    GJAPI.DataStoreGetKeys = function (iStore, pCallback) {
-      // send get-keys request
+    /**
+     * Send get-keys request
+     * @param {number} iStore
+     * @param {function} pCallback
+     */
+    GJAPI.DataStoreGetKeys = (iStore, pCallback) => {
       GJAPI.SendRequest("/data-store/get-keys/", iStore, pCallback);
     };
 
-    // ****************************************************************
-    // create asynchronous request
+    /**
+     * Create asynchronous request
+     * @param {string} sUrl
+     * @param {string} sBodyData
+     * @param {function} pCallback
+     */
     function __CreateAjax(sUrl, sBodyData, pCallback) {
       if (typeof sBodyData !== "string") sBodyData = "";
 
       Scratch.canFetch(sUrl).then((allowed) => {
         if (!allowed) {
+          pCallback("TODO: Put a message here");
```

Comment:
Currently this will never actually run, but in theory it could so it needs error handling. I don't know what message to put there.